charts/prometheus/README.md

```diff
@@ -61,9 +61,13 @@ helm upgrade [RELEASE_NAME] [CHART] --install
 
 _See [helm upgrade](https://helm.sh/docs/helm/helm_upgrade/) for command documentation._
 
+## Updating values.schema.json
+
+A [`values.schema.json`](https://helm.sh/docs/topics/charts/#schema-files) file has been added to validate chart values. When `values.yaml` file has a structure update (i.e. add a new field, change value type, etc.), modify `values.schema.json` file manually or run `helm schema-gen values.yaml > values.schema.json` to ensure the schema is aligned with the latest values. Refer to [helm plugin `helm-schema-gen`](https://github.com/karuppiah7890/helm-schema-gen) for plugin installation instructions.
+
 ### To 19.0
 
-Prometheus has been updated to version v2.40.5.
+Prometheus has been updated to version v2.41.0.
```

Comment:
```suggestion
Prometheus has been updated to version v2.40.5.
```

Reply:
I have reverted this back. This update was not relevant to the json validation I did. I noticed the appVersion is now v2.41.0 for chart version v19. Is v2.40.5 pined for v19.0 since this is the earliest appVersion for v19.0?